tools/build_cros_gcc

```diff
@@ -133,6 +143,11 @@
  ./$MPC.tar.gz 
 
  cd $GCC
+ GCC_PATCHES="$GCC-$ISL.patch"
+ for patch in $GCC_PATCHES; do
+   wget https://github.com/JL2210/patches/raw/master/$patch.patch
+   patch -p1 < $patch.patch
```

Comment:
I think you will want `$patch` instead of `$patch.patch`.  Please test. :)

Reply:
Oops. I don't have that kind of Internet connection.